client/src/components/Menu/index.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import withStyles from 'isomorphic-style-loader/withStyles';
+import classNames from 'classnames';
+import Hamburger from 'react-hamburger-menu';
+import MenuItem from './MenuItem';
+
+import s from './Menu.css';
+import dashboardIcon from '../../assets/dashboard.svg';
+import temperatureIcon from '../../assets/temp.svg';
+import windIcon from '../../assets/wind.svg';
+import qualityIcon from '../../assets/quality.svg';
+import websiteIcon from '../../assets/website.svg';
+import facebookIcon from '../../assets/facebook.svg';
```

Comment:
hmmm, are all these assets globally needed? You could place them in the component's directory

Reply:
I consider good practice to place all images under assets folder, isnt it right?